### PR TITLE
feat: assets revision atomic roles

### PR DESCRIPTION
**Pull request by gabrielfs7**
Assets Revision Atomic Roles:

https://oat-sa.atlassian.net/browse/ADF-326

### CODE REVIEW

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: migrations/Version202107270944252141_taoRevision.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+namespace oat\taoRevision\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\taoRevision\model\user\TaoRevisionRoles;
+use oat\tao\scripts\update\OntologyUpdater;
+use oat\tao\scripts\tools\accessControl\SetRolesAccess;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+
+final class Version202107270944252141_taoRevision extends AbstractMigration
```
Comment (gabrielfs7) [inline migrations/Version202107270944252141_taoRevision.php:11]: I probably wont need this migration

Comment (gabrielfs7) [reply migrations/Version202107270944252141_taoRevision.php:11]: No, I need :)

File: scripts/install/SetupRolesPermissions.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+ */
+
+namespace oat\taoRevision\scripts\install;
+
+use oat\oatbox\extension\InstallAction;
+use oat\tao\scripts\tools\accessControl\SetRolesAccess;
+use oat\tao\scripts\update\OntologyUpdater;
+use oat\taoRevision\model\user\TaoRevisionRoles;
+
+class SetupRolesPermissions extends InstallAction
```
Comment (shpran) [inline scripts/install/SetupRolesPermissions.php:28]: You don't need this script
ACL rules will be installed from `manifest.php->acl`

File: manifest.php [hunk 3]
```diff
@@ -35,10 +36,41 @@
...
+            ],
+        ],
     ],
     'install' => [
+        'rdf' => [
+            __DIR__ . '/model/ontology/tao-revision.rdf',
+        ],
         'php' => [
             SetupRevisions::class,
+            SetupRolesPermissions::class,
```
Comment (shpran) [inline manifest.php:73]: https://github.com/oat-sa/extension-tao-revision/pull/127/files#r679208738

**Review by chinnu-focaloid — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful

**Comment by AntonNeviarouski** [comment]
QA verified on http://test-gabriel.playground.kitchen.it.taocloud.org:42221/

File: scripts/install/SetupRolesPermissions.php (hunk 2 above)
Comment (gabrielfs7) [reply scripts/install/SetupRolesPermissions.php]: Removed

**Pull request merged**